kotlin-checks-test-sources/build.gradle.kts

```diff
@@ -19,6 +19,7 @@ dependencies {
     implementation("com.google.guava:guava:30.1.1-jre")
     implementation("org.apache.commons:commons-lang3:3.9")
     implementation("commons-lang:commons-lang:2.6")
+    implementation("org.jetbrains.kotlin:kotlin-script-runtime:1.4.10")
```

Comment:
Why such an old version? We use Kotlin 1.8

Reply:
Also, if we move these files out from the sources, we don't need this dependency